### PR TITLE
Lesson 1.3

### PR DESCRIPTION
**Pull request by Maarimar**
Moved List Item into New "Todo List Item" Component, Handle "Add Todo Form" Submit, e "New Todo" in React State

### CODE REVIEW

File: src/TodoListItem.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+
+const TodoListItem = (props) =>{
+    return(
+<> <li key={props.id}>
```
Comment (alenamedved) [inline src/TodoListItem.js]: <> fragment is not needed here, return li item

File: src/TodoList.js [hunk 2]
```diff
@@ -1,15 +1,14 @@
 import React from "react";
+import TodoListItem from "./TodoListItem";
 const todoList =[{id: 1, tittle: 'Grocery shopping'}, {id: 2, tittle:'pack boxes'}, {id:3, tittle:'buy headphones'}];
 
 function TodoList() {
   return (
     <div>
     <ul>
-      {todoList.map((el)=>{
```
Comment (alenamedved) [inline src/TodoList.js]: same, you can remove a div element, just return a ul

**Review by alenamedved — COMMENTED** [review]
code looks good, don't forget to format it
great job @Maarimar

File: src/TodoList.js (hunk 2 above)
Comment (Maarimar) [reply src/TodoList.js]: I don't exactly understand what you mean.  Do you mean remove the div and only return de ul?

Comment (alenamedved) [reply src/TodoList.js]: yes,sorry, just return an ul

**Pull request merged**